README.md

```diff
@@ -3,544 +3,66 @@
 [![Goreport status](https://goreportcard.com/badge/github.com/flashbots/suave-geth)](https://goreportcard.com/report/github.com/flashbots/suave-geth)
 [![CI status](https://github.com/flashbots/suave-geth/workflows/Checks/badge.svg?branch=main)](https://github.com/flashbots/suave-geth/actions/workflows/checks.yml)
 
-[SUAVE](https://writings.flashbots.net/mevm-suave-centauri-and-beyond) is designed to decentralize the MEV supply chain by enabling centralized infrastructure (builders, relays, centralized RFQ routing, etc.) to be programmed as smart contracts on a decentralized blockchain.
+[SUAVE](https://writings.flashbots.net/the-future-of-mev-is-suave) is designed to keep the benefits of MEV well distributed by enabling centralized infrastructure (builders, relays, centralized RFQ routing, etc.) to be programmed as smart contracts on a decentralized blockchain.
```

Comment:
Hmm I'm not sure about the new intro sentence. We've been talking about the MEV supply chain for quite some time, and "to keep the benefits of MEV well distributed by" seems to me to not quite capture the full picture.